packages/runtime/src/runtimes/react/react-runtime.ts

```diff
@@ -9,6 +9,15 @@ import type { ComponentType } from '../../state/react-page'
 
 import { RuntimeCore } from './runtime-core'
 
+function validateComponentType(type: string, component?: ComponentType): void {
+  const componentName = component?.name ?? 'Component'
+  if (typeof type !== 'string') {
```

Comment:
Do we want to check that the string is not empty as well?

Reply:
I think we probably should disallow empty strings, yeah. A user would probably not want to do that intentionally

Reply:
Addressed